### ISSUE TITLE
Move "Troubleshoot" out of About and into Settings as its own nav item

### ISSUE THREAD

**Issue by shawnborton** [Expensify/App#38594]
We want to take the Troubleshoot page:
![CleanShot 2024-03-19 at 10 19 04@2x](https://github.com/Expensify/App/assets/2319350/d563b79f-d9c5-4c0e-8f42-2d251f8de802)

And move it into the Settings navigation as it's own nav item, right above the Sign Out option at the bottom. When doing this, the Troubleshoot UI should look like the About page, where we use the card style and a nice header with an illustration + h2:
![image](https://github.com/Expensify/App/assets/2319350/5d6301fd-f966-4ea3-bf7c-f97451f84a20)

cc @TMisiukiewicz since I think you originally implemented this? This would just be a follow up to that. 

cc @muttmuure @Expensify/design

**Comment by godofoutcasts94** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Move "Troubleshoot" out of About and into Settings as its own nav item
### What is the root cause of that problem?
The `Troubleshoot` is inside `About Page` i.e. here : 
https://github.com/Expensify/App/blob/71e3aa4eb3be90eeadd27260dfb87a3ff67185be/src/pages/settings/AboutPage/AboutPage.tsx#L90-L94
### What changes do you think we should make in order to solve the problem?

We need to move it into `InitialPage`, just above `Signout button` here : 
https://github.com/Expensify/App/blob/71e3aa4eb3be90eeadd27260dfb87a3ff67185be/src/pages/settings/InitialSettingsPage.tsx#L250-L256
### What alternative solutions did you explore? (Optional)
N/A

### Result :

<img width="1470" alt="Screenshot 2024-03-19 at 9 50 24 PM" src="https://github.com/Expensify/App/assets/158435970/da338af8-ba39-4dee-b261-25555d8b55f3">

**Comment by godofoutcasts94** [comment]
Upated [Proposal](https://github.com/Expensify/App/issues/38594#issuecomment-2007614043), will add updated screenshot in sometime

**Comment by muttmuure** [comment]
I think we should actually make this an External issue, since Tomasz is working on a higher value performance project

**Comment by godofoutcasts94** [comment]
Hey @muttmuure, I have already posted a proposal and I am already working. Should I complete it and post a test branch ?

**Comment by godofoutcasts94** [comment]
Updated [Proposal](https://github.com/Expensify/App/issues/38594#issuecomment-2007614043) since in the initial comment it was asked to be assigned to  
<img width="840" alt="Screenshot 2024-03-21 at 6 18 02 PM" src="https://github.com/Expensify/App/assets/158435970/bf424d84-9de1-4fda-a128-c7d269e4cc5e">

**Comment by godofoutcasts94** [comment]
> ## Proposal
> Updated due to people posting complete diff for no reason (which is discouraged), and this is not a good practice.

In the original proposal template it is mentioned not to propose diffs but still folks are doing it.
<img width="1117" alt="Screenshot 2024-03-21 at 6 22 01 PM" src="https://github.com/Expensify/App/assets/158435970/6e41fbd8-a4ee-423a-a6c6-44942af78a02">

**Comment by alitoshmatov** [comment]
Thank you everyone for your proposal, I see all the proposals are similar in some way. There are a lot of updating in all proposals which is confusing, I try to be as fair as possible

**Comment by alitoshmatov** [comment]
@godofoutcasts94 I see you have a problem in section subtitle where there should be link to submit a bug

**Comment by godofoutcasts94** [comment]
> @godofoutcasts94 I see you have a problem in section subtitle where there should be link to submit a bug

I will have a look and rectify it maybe I have missed in the proposal. Will fix it in the proposal and get back to you

**Comment by alitoshmatov** [comment]
@shubham1206agra Your proposal looks good, same approach as other proposals except you have opted out section subtitle, and just put text inside section which is why its style is a little bit different.

**Comment by alitoshmatov** [comment]
@ShridharGoel Your proposal also looks good, and almost the same as @shubham1206agra 's proposal

**Comment by alitoshmatov** [comment]
@Krishna2323 Your proposal is also similar to others except it lacks some details like how you handle section subtitle with a link

**Comment by alitoshmatov** [comment]
@tienifr Your proposal is also almost the same as others. One difference you approached section subtitle a little bit differently by adding, renderSubtitle.

**Comment by alitoshmatov** [comment]
Thank you everyone. To simplify and move forward, I narrowed down two potential proposals. 
@shubham1206agra and @tienifr 's proposals look good and provide potential solution to section subtitle which should have a link in it. I see both of your point and note sure which is better. Would love to hear from you some arguments and details related to this point of your solution, any advantages, any similar use cases would help. Look forward for your analysis

**Comment by godofoutcasts94** [comment]
so how about my proposal? @alitoshmatov, I can fix and come back to you, still my proposal won't be considered ?

**Comment by alitoshmatov** [comment]
@godofoutcasts94 If you have different approach to the problem we are facing with section subtitle, I would be happy to consider your proposal.

**Comment by godofoutcasts94** [comment]
Updated test branch

**Comment by alitoshmatov** [comment]
Thank you everyone for your proposal I understand that section subtitle might seem small detail but I think it is important enough that we should approach it with ideal solution, also all proposal being almost the same and a lot of edits made the review process difficult. I tried to be as reasonable as possible when choosing proposal.

**Comment by alitoshmatov** [comment]
I do think that `renderSubtitle` suggested by @tienifr is the correct approach and I agree with their [points](https://github.com/Expensify/App/issues/38594#issuecomment-2017663945). It might seem one-off thing but as soon as we need another similar case where section subtitle require more than just a plain text we run into problem, that is why for the sake of future proofing we should apply this solution.

We can go with @tienifr 's [proposal](https://github.com/Expensify/App/issues/38594#issuecomment-2012104406)

C+ reviewed 🎀 👀 🎀

**Comment by danieldoglas** [comment]
big discussions on this one. Thanks for curating it all @alitoshmatov.

@shawnborton, as @tienifr pointed, can you please send us the lightbulb illustration?

> We need to supply a Lightbulb illustration as the icon in the Header, currently we don't have that in the app so I temporarily substitute by the black light bulb icon

assigning @tienifr

**Comment by shawnborton** [comment]
@tienifr looks like we have a small regression on iOS - notice how the illustration is sitting too far down compared to desktop?

iOS:
![CleanShot 2024-04-18 at 06 05 54@2x](https://github.com/Expensify/App/assets/2319350/19b74d8f-7e02-489a-929f-7c1d778da8c9)

Desktop:
![CleanShot 2024-04-18 at 06 06 22@2x](https://github.com/Expensify/App/assets/2319350/1ea4fa55-088c-4892-8907-88c3b93daec2)

Can you please submit a follow up PR to fix?

**Comment by shawnborton** [comment]
Friendly bump @tienifr

**Comment by alitoshmatov** [comment]
> @tienifr can you confirm if this is really related? https://github.com/Expensify/App/issues/41275

This is not related

**Comment by shawnborton** [comment]
@tienifr what is the latest with adjusting the illustration position as a follow up?

**Comment by shawnborton** [comment]
We still have an outstanding regression that needs to be taken care of, take a look at [this comment](https://github.com/Expensify/App/issues/38594#issuecomment-2063503322)

**Comment by dannymcclain** [comment]
Ah I see! Thanks! I see the follow-up PR for that is in the works.

**Comment by alitoshmatov** [comment]
### Regression Test Proposal

1. Go to settings
2. Check `Troubleshoot` menu item is there in general section after `About` menu item
3. Click on `Troubleshoot`
4. Check that the page is exactly like given in the design guidelines
5. Make sure everything in `Troubleshoot` page is working in all platforms

Do we agree 👍 or 👎

**Comment by alitoshmatov** [comment]
@muttmuure This one is ready for payment

**Comment by shawnborton** [comment]
I found another regression with this one: the Troubleshoot page uses white/light text for the status bar text, and thus you can't see the status bar on iOS:
![CleanShot 2024-06-19 at 15 55 01@2x](https://github.com/Expensify/App/assets/2319350/566d8d8e-1ccf-4609-b232-709b9d9ba189)

@tienifr @alitoshmatov can you have a look at that please? Thanks!

**Comment by shawnborton** [comment]
Exactly, check every other page in Settings and you will see the expected behavior.

**Comment by alitoshmatov** [comment]
https://github.com/Expensify/App/pull/44048 PR was merged and now on staging

**Comment by alitoshmatov** [comment]
Regression test is [here](https://github.com/Expensify/App/issues/38594#issuecomment-2178379393)

**Comment by alitoshmatov** [comment]
Regression period ends on 17th, pr was [merged](https://github.com/Expensify/App/pull/44048#issuecomment-2220200454) on 10th

**Comment by muttmuure** [comment]
$500 - @alitoshmatov 
$500 - @tienifr

**Comment by muttmuure** [comment]
Paid
